### ISSUE TITLE
`no_implicit_braces` false positive on multi-line expression

### ISSUE THREAD

**Issue by aseemk** [clutchski/coffeelint#459]
This passes:

```coffee
list =
    count: 10
    items:
        for item in items
            name: item.Name
            age: item.Age
```

But this fails:

```coffee
list =
    count: 10
    items:
        for item in items
            if not item
                throw new Error 'Unexpected: falsy item in list!'

            name: item.Name
            age: item.Age
```

On the `name: item.Name` line with a `no_implicit_braces` error (with `strict` set to `false`).

I can work around by adding braces, but it'd be nice to retain implicit braces in this multi-line case. =)
